src/backends/backend/triton_model_instance.cc

```diff
@@ -351,20 +367,22 @@ TritonModelInstance::TritonBackendThread::Enqueue(
 }
 
 void
-TritonModelInstance::TritonBackendThread::BackendThread(const int nice)
+TritonModelInstance::TritonBackendThread::BackendThread(
+    const int nice, const int32_t device_id)
 {
 #ifndef _WIN32
   if (setpriority(PRIO_PROCESS, syscall(SYS_gettid), nice) == 0) {
     LOG_VERBOSE(1) << "Starting backend thread for " << name_ << " at nice "
-                   << nice << "...";
+                   << nice << " on device " << device_id << "...";
```

Comment:
You can leave this "nice" logic in here since it may be useful in the future. But for now always set nice == 0.